state_machine/unit_tests/test_commands.py

```diff
@@ -27,14 +31,26 @@ async def test_query(self):
     async def test_tq_len(self):
         await self.cmd_test(cdh.TQ_SIZE, b'', '0')
 
-    async def cmd_test(self, cmd, args, expected):
+    async def test_tq_clear(self):
+        # clog tx queue
+        await send_cmd(cdh.EXEC_PY, b"for _ in range(10):\n\ttq.push(Message(5, b'hello'))")
+
+        # clear the tx queue
+        for _ in range(radio.TX_SKIP):
```

Comment:
Why are you using TX_SKIP here? Shouldn't this take a maximum of two calls to send_cmd?

Reply:
No, the tx queue is clogged, so the satelitte transmits a couple times before the duty cycle forces it to receive